Foodhut/src/styles/sections/introduction.css

```diff
@@ -73,43 +73,44 @@
 }
 
 .search-black-icon {
-  padding: 15px 12px 15px 15px;
-  top: 50%;
+  background-image: url('../../assets/images/search-icon-black.svg');
+  width: 24px;
+  height: 24px;
+  margin: 15px;
```

Comment:
Better to use auto as .search-white-icon

Reply:
i fixed it